client/src/Components/LogNavBar.js

```diff
@@ -4,7 +4,16 @@ import { AppBar, Toolbar, Button } from "@material-ui/core";
 import Searchbar from "./Searchbar";
 import Logo from "./Logo";
 
-function LogNavBar() {
+function LogNavBar(props) {
+  const handleLogout = e => {
```

Comment:
Okay so I see here you've declared the logout functionality, but haven't used an `onClick` event to tie it to the logout button on Line 26. This won't work unless you pass the Logout Button a `onClick` value, something like this: `onClick={() => handleLogout() }`